Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/BlendGraphWidget.h

```diff
@@ -37,20 +37,29 @@ namespace EMStudio
     class BlendGraphMimeEvent : public GraphCanvas::GraphCanvasMimeEvent
     {
     public:
-        explicit BlendGraphMimeEvent(QString typeString, QString namePrefix);
+        AZ_RTTI(BlendGraphMimeEvent, "{AA7C8960-C7BA-4F26-B52B-97CF4AC3CB39}", GraphCanvas::GraphCanvasMimeEvent);
+        AZ_CLASS_ALLOCATOR(BlendGraphMimeEvent, AZ::SystemAllocator, 0);
+        static void Reflect(AZ::ReflectContext* context);
+
+        BlendGraphMimeEvent() = default;
+        explicit BlendGraphMimeEvent(const AZStd::string_view typeString, const AZStd::string_view namePrefix);
```

Comment:
nit: We can drop the explicit here as we have two arguments being passed to the constructor making implicit conversions a lot less likely

Reply:
Done